bats/foreman_helper.bash

```diff
@@ -45,6 +51,12 @@ tSkipIfPulp2() {
   fi
 }
 
+tSkipIfNoPulp3() {
```

Comment:
Is this redundant with the `tSkipIfOlderThan41` function? When I ran the bats, they got skipped due to this evaluating to true. Not sure where `tIsPulpcore` is defined in any case

Reply:
I got the tests running by removing the references to this function btw

Reply:
@jeremylenz this still needs attention

Reply:
Ah, good catch!

Reply:
@jturel updated!